src/backend/src/utils/utils.ts

```diff
@@ -23,3 +23,9 @@ export const validateInputs = (req: Request, res: Response, next: Function): Res
   }
   next();
 };
+
+export const transformDate = (date: Date) => {
+  const month = date.getMonth() + 1 < 10 ? `0${date.getMonth() + 1}` : (date.getMonth() + 1).toString();
+  const day = date.getDate() < 10 ? `0${date.getDate()}` : date.getDate().toString();
+  return `${date.getFullYear().toString()}-${month}-${day}`;
+};
```

Comment:
ok don't know how I didn't catch this before, but we should not have a file called utils.ts in a folder called utils, just kinda redundant. Pls move transform date out of here into another file (we should probably have a datetime utils file in the backend as well), I'll make another ticket to remove this file since it's gonna cause a lot of import changes, dw hold this PR up